### PR TITLE
Add .envrc

### PR DESCRIPTION
**Pull request by qgp**

### CODE REVIEW

File: .envrc [hunk 1]
```diff
@@ -0,0 +1,15 @@
+if [[ `hostname` = "alicecerno2" ]] && [[ -z ${ROOT_BASEDIR} ]]; then
```
Comment (vkucera) [inline .envrc]:
```suggestion
if [[ $(hostname) == "alicecerno2" ]] && [[ -z ${ROOT_BASEDIR} ]]; then
```

File: .envrc [hunk 2]
```diff
@@ -0,0 +1,15 @@
...
+	ROOT_VERSION=v6-32-04
+	ROOUNFOLD_VERSION=2.0.1
+
+	PREFIX=/home/pyadmin/software_mlhep
+	layout python ${PREFIX}/install/pyenv/versions/${PYTHON_VERSION}/bin/python3
+	path_add PYTHONPATH ${PREFIX}/install/root-${ROOT_VERSION}_py-${PYTHON_VERSION}/lib
+	PATH_add ${PREFIX}/install/root-${ROOT_VERSION}_py-${PYTHON_VERSION}/bin
+	path_add LD_LIBRARY_PATH ${PREFIX}/install/root-${ROOT_VERSION}_py-${PYTHON_VERSION}/lib
+	# path_add LD_LIBRARY_PATH ${PREFIX}/install/RooUnfold-${ROOUNFOLD_VERSION}_root-${ROOT_VERSION}_py-${PYTHON_VERSION}/lib
+	path_add LD_LIBRARY_PATH ${PREFIX}/build/RooUnfold-${ROOUNFOLD_VERSION}_root-${ROOT_VERSION}_py-${PYTHON_VERSION}
```
Comment (vkucera) [inline .envrc:13]: Is this correct?

Comment (qgp) [reply .envrc:13]: Yes, RooUnfold 2.0.1 does not install

File: .envrc (hunk 1 above)
Comment (qgp) [reply .envrc]: What's the benefit?

Comment (vkucera) [reply .envrc]: https://www.shellcheck.net/wiki/SC2006

Comment (qgp) [reply .envrc]: Ok, I'm old 😳

Comment (vkucera) [reply .envrc]: Equality check inside `[[ ]]` should use `==`.

**Pull request merged**